tests/BOAClient.test.ts

```diff
@@ -438,4 +438,23 @@ describe ('BOA Client', () =>
 
         doneIt();
     });
+
+    it ('test for getHeightAt', (doneIt: () => void) =>
+    {
+        // Set URL
+        let uri = URI("http://localhost").port(port);
+        // Create BOA Client
+        let boa_client = new boasdk.BOAClient(uri.toString());
+        let date = new Date(Date.UTC(2020,3,29,0,0,0));
```

Comment:
```suggestion
        let date = new Date(Date.UTC(2020, 3, 29, 0, 0, 0));
```
I think it would be better to add a space.

Reply:
Applied a code style.